exercises/rotational-cipher/.meta/hints.md

```diff
@@ -1,3 +1,10 @@
+## Bonus Tasks
+
+This exercise is quite large and potentially complicated, and that makes it more challenging
+and time-consuming to mentor. To help your mentor out, please don't submit code for the
+bonus exercises until your mentor has reviewed your solution for the first part of the
+exercise.
```

Comment:
```suggestion
This exercise is quite large and potentially complicated, and that makes it more challenging
and time-consuming to mentor. If you are in Mentor Mode or are intending to request a mentor in Practice model, to help your mentor out, please don't submit code for the
bonus exercises until your mentor has reviewed your solution for the first part of the
exercise.
```

Reply:
Thanks! I'll also change that hint in circular-buffer

Reply:
Sweet :)